main.tf

```diff
@@ -4,17 +4,17 @@ provider "aws" {
 
 resource "aws_instance" "webserver" {
   ami           = "ami-01e36b7901e884a10"
-  instance_type = "t2.micro"
+  instance_type = "t2.large"
 
   tags = {
     Name = "webserver"
   }
 }
 
 resource "aws_s3_bucket" "accuricsbucketdemo" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure at-rest server side encryption (SSE) is enabled for AWS S3 Buckets**
 
**Vulnerability**
AWS S3 Buckets do not have at-rest server side encryption enabled to protect sensitive data.
 
**Remediation**
On the AWS S3 console, for each bucket that failed the rule, navigate to the Properties tab, and select Default encryption. Select an encryption method (AES-256 or AWS-KMS).

source: accurics-stageone